src/utils/ElementalReaction.js

```diff
@@ -0,0 +1,27 @@
+import { keys } from "lodash";
+/**
+ * @description Calculates the melt/vaporize multiplier for a character
+ * @param {string} damageElement - a valid element
+ * @param {string} statusElement -another valid element
+ * @returns
+ */
+export default function ElementalReactionMultiplier(
+  elementalMastery,
+  damageElement,
+  statusElement
+) {
+  const elementInteract = damageElement + "->" + statusElement;
+  const reactionMap = {
+    "Pyro->Cryo": 2,
+    "Cyro->Pyro": 1.5,
+    "Hydro->Pyro": 2,
+    "Pyro->Hydro": 1.5,
+  };
+  const reactionMultiplier = reactionMap[elementInteract];
+  if (!!reactionMultiplier) return 1;
```

Comment:
typo? im not sure why you would need 2 exclamation marks and also it would make sense if you return 1 if !reactionMultiplier